packages/elasticsearch-asset-apis/src/base-api/index.ts

```diff
@@ -516,3 +519,11 @@ function divideKeyArray(keysArray: string[], num: number): string[][] {
 
     return results;
 }
+
+function indexMatcher(index: string): (input: string) => boolean {
+    if (isWildCardString(index)) {
+        return (indexVal) => matchWildcard(index, indexVal);
+    }
+
+    return (indexVal) => indexVal.match(index) != null;
```

Comment:
Since it is not a wildcard string, we should includes not match to be more explicit